client/components/promo-section/promo-card/price.tsx

```diff
@@ -1,14 +1,19 @@
 import classnames from 'classnames';
 import { useTranslate, TranslateResult } from 'i18n-calypso';
-import { FunctionComponent } from 'react';
+import { FunctionComponent, ReactElement } from 'react';
```

Comment:
Can you only import the `ReactElement` type?
```suggestion
import { FunctionComponent } from 'react';
import type { ReactElement } from 'react';
```